### ISSUE TITLE
Advertise S3 solution

### ISSUE THREAD

**Issue by mrocklin** [coiled/feedback#3]
In transitioning users from laptop workflows to cloud workflows we're going to need to get them more comfortable with moving data to S3.  I see three parts to this:

- [ ] Point them to s3fs and give them a simple example in documentation
- [ ] Give organizations a shared bucket of scratch space
- [ ] Make sure that their local session has sufficient permissions to upload data.  This maybe means making an IAM role for them, and injecting those credentials into their local boto3 session.

    Alternatively, we proxy this upload through the `Cloud` constructor and post to S3 ourselves.  This is error prone though because they may then want to access that data from pandas locally as well.

**Comment by tdliu** [comment]
+1. Was just about to post something along these lines.

**Comment by mrocklin** [comment]
Right.  We take on a bit more complexity by not requiring that computations are submitted from the platform.  This requires some care to do well, but I don't think should be particularly onerous.

**Comment by shughes-uk** [comment]
stale

**Comment by ntabris** [comment]
I think this is still with consideration.
